engine/frontend/src/component/Home.js

```diff
@@ -39,6 +38,16 @@ const Home = () => {
     const [enclaveLoading, setEnclaveLoading] = useState(false)
     const {appData, setAppData} = useAppContext()
 
+    const handleDeleteClick = (enclaveName) => {
+        console.log("name ", enclaveName)
+        const filteredEnclaves = enclaves.filter(enclave => enclave.name !== enclaveName)
+        const makeRequest = async () => {
+            await removeEnclave(appData.jwtToken, appData.apiHost, enclaveName)
```

Comment:
@Peeeekay can we listen to the response and determine if the removal was successful or not?